src/markdown-pages/collect-data/query-data-nrql.mdx

```diff
@@ -9,10 +9,13 @@ tileShorthand:
   description: 'Query default event data, custom events, and attributes'
 redirects:
   - /technology/nrql
+resources:
+  - title: 'New Relic One NRQL Lessons'
```

Comment:
Similar to above, just make it 'NRQL lessons' - lower case l in lessons as well.

Reply:
you are still getting me on sentence case :) some day.. i'll stop this bad habit.